lib/service/storage/elasticsearch.ts

```diff
@@ -1966,12 +2099,19 @@ class ElasticSearch extends Service {
    *
    * @returns {Promise.<String[]>}
    */
-  async deleteIndexes(indexes = []) {
+  async deleteIndexes(indexes = []): Promise<string[]> {
     if (indexes.length === 0) {
       return Bluebird.resolve([]);
     }
-    const deleted = new Set();
-
+    const deleted = new Set<string>();
+    for (const index of indexes) {
+      if (this.virtualIndex.isVirtual(index)) {
+        //TODO : remove second await?
```

Comment:
You can leave it, actually this is really not a critical part of the code since this action is not used very often

Reply:
ok